ytfzf

```diff
@@ -704,6 +708,21 @@ scrape_yt () {
 	unset videos_json yt_json yt_html
 }
 
+parse_special_search () {
+    search="$1"
+    number="$(printf "%s" "$search" | grep -Po "[\-0-9]+")"
```

Comment:
since we already know that the string satisfies the regex is it possible to trim of the `!` to get the number? Maybe with a parameter expansion with `${var#!}`